libraries/AP_Mount/AP_Mount_Backend.h

```diff
@@ -82,6 +82,9 @@ class AP_Mount_Backend
     // send a GIMBAL_REPORT message to the GCS
     virtual void send_gimbal_report(const mavlink_channel_t chan) {}
 
+    // handle a GLOBAL_POSITION_INT message
+    virtual bool handle_global_position_int(uint8_t msg_sysid, const mavlink_global_position_int_t &packet);
```

Comment:
This doesn't currently need to be virtual; we can save some vmdt space omitting it until needed.

Reply:
ok